app/gui/view/graph-editor/src/component/node/input/widget.rs

```diff
@@ -1651,7 +1651,7 @@ impl<'a> TreeBuilder<'a> {
         let ptr_usage = self.pointer_usage.entry(main_ptr).or_default();
         let widget_id = main_ptr.to_identity(ptr_usage);
 
-        let is_placeholder = span_node.is_expected_argument();
+        let is_placeholder = span_node.is_expected_argument() || span_node.is_expected_operand();
```

Comment:
Looks like the same change also needs to be made in `list_editor.rs`. Maybe there should be a `is_placeholder` method on the span node?